editor/export/editor_export_platform.cpp

```diff
@@ -637,10 +655,20 @@ bool EditorExportPlatform::_export_customize_object(Object *p_object, LocalVecto
 	return changed;
 }
 
+bool EditorExportPlatform::_is_editable_ancestor(Node *p_root, Node *p_node) {
+	while (p_node != nullptr) {
```

Comment:
This should fix the aforementioned spam:

```suggestion
	while (p_node != nullptr && p_node != p_root) {
```